src/ds++/test/tstPacking_Utils.cc

```diff
@@ -305,10 +304,10 @@ void packing_test(rtt_dsxx::UnitTest &ut) {
     u.extract(2, cc);
 
     for (size_t i = 0; i < 50; ++i)
-      if (x[i] != 0)
+      if (!rtt_dsxx::soft_equiv(x[i], 0.0))
```

Comment:
Do you want to use `x[i] < numeric_limits<double>::min()` for comparisons to zero as is done elsewhere in this PR?